### ISSUE TITLE
Root entry is missing in Breadcrumb

### ISSUE THREAD

**Issue by tumidi** [innodoc/innodoc-webapp#66]
Root/Home entry is not there.

Right now it looks like:

```
Section 1 > Section 1.1 > Section 1.2
```

Instead it should look like:

```
Home > Section 1 > Section 1.1 > Section 1.2
```

(where Home is obviously a link to course TOC page .)

See original issue #21

**Comment by tumidi** [comment]
In GitLab by @ghost on Oct 25, 2018, 16:05

created branch [`51-root-entry-is-missing-in-breadcrumb`](https://gitlab.tubit.tu-berlin.de/innodoc/innodoc-webapp/compare/master...51-root-entry-is-missing-in-breadcrumb)

 *By Dennis Otrombowsky on 2018-10-25T14:05:37 (imported from GitLab)*

**Comment by tumidi** [comment]
In GitLab by @ghost on Oct 27, 2018, 21:21

mentioned in commit f3aa7696e97ef8ae391e513d408786ca5b7a71e3

 *By Dennis Otrombowsky on 2018-10-27T19:21:23 (imported from GitLab)*

**Comment by tumidi** [comment]
mentioned in commit 62f18902db957f7f6beef270adbaf6fd63f92271

**Comment by tumidi** [comment]
closed via commit 62f18902db957f7f6beef270adbaf6fd63f92271

**Comment by tumidi** [comment]
closed via commit bfa0f5288659726b003bda42477301c15463033d

**Comment by tumidi** [comment]
mentioned in commit bfa0f5288659726b003bda42477301c15463033d
